src/containers/genePage/genomeFeatureViewer.js

```diff
@@ -45,7 +44,8 @@ class GenomeFeatureViewer extends Component {
     }
 
     for (let name in names) {
-      trackDataWithHighlight = trackDataWithHighlight + (name == 0 ? '?' : '&') + 'name=' + names[name];
+      let encodedName = names[name].replace('\|','%7C');
```

Comment:
Perhaps [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) would be a more general solution here?

Reply:
agreed